Gemfile

```diff
@@ -19,6 +19,10 @@ gem 'jbuilder', '~> 2.5'
 group :development, :test do
   gem 'pry'
   gem 'travis'
+  gem 'rspec-pride'
+  gem 'orderly'
+  gem 'factory_bot_rails'
+  gem 'faker'
```

Comment:
This is actually more for my understanding, do these testing gems belong in the :development :test group or strictly just in the :test group below?